src/function.ts

```diff
@@ -179,7 +181,9 @@ export class lf {
       const cloudFrontId = req.header(HttpHeaderAmazon.CloudfrontId);
       const traceId = req.header(HttpHeaderAmazon.TraceId);
       const lambdaId = context.awsRequestId;
-      req.set('aws', { cloudFrontId, traceId, lambdaId });
+      if (cloudFrontId || traceId || lambdaId) {
```

Comment:
if any of them exists 
or 
if all of them exists here?

Reply:
if any of them exists. saves us writing a `aws: {}` into the logs when none of them exist